sample/src/main/res/values/strings.xml

```diff
@@ -345,4 +346,18 @@
     <string name="info_message_example_2_button">Do something</string>
     <string name="info_message_example_3_headline">Notice message</string>
 
+    <!--  Menu Panel Row View  -->
+    <string name="menu_panel_placeholder_title">Title</string>
+    <string name="menu_panel_placeholder_body">Body</string>
+    <string name="menu_panel_example_1_title">Pay As You Earn (PAYE)</string>
+    <string name="menu_panel_example_1_body">Check your tax codes and Income Tax from PAYE income sources</string>
+    <string name="menu_panel_example_2_title">Messages</string>
+    <string name="menu_panel_example_2_body">Messages and letters from HMRC</string>
+    <string name="menu_panel_example_3_title">Help to Save account</string>
+    <string name="menu_panel_example_3_body">A savings account with bonuses designed to help you start saving</string>
+    <string name="menu_panel_example_4_title">News</string>
+    <string name="menu_panel_example_4_body">News and current tax related information from HMRC</string>
+    <string name="menu_panel_example_5_title">Long Long Long Long Long Long Long Long Long Long Long Long Long Long Long Long Title</string>
+    <string name="menu_panel_example_5_body">Really Really Really Really Really Really Really Really Really Long Body</string>
```

Comment:
Instead of defining these to provide long text examples, you can use our pre-existing "long_text" and "longer_text" strings that are at the top of this file.